app/safe/internal/bootstrap/k8s.go

```diff
@@ -13,6 +13,7 @@ package bootstrap
 import (
 	"context"
 	"github.com/pkg/errors"
+	"github.com/vmware-tanzu/secrets-manager/core/log"
```

Comment:
@abhishek44sharma can you also update the documentation under the `./docs` folder? 

https://vsecm.com/docs/configuration/

^ this should include the new environment variable to be consistent.

Reply:
Done